### ISSUE TITLE
Should bar charts (in horizontal orientation along the braille display) be supported?

### ISSUE THREAD

**Issue by jasonjgw** [fizzstudio/sparkbraille#6]
This is an idea that we discussed briefly at the Code Sprint. The purpose
of this issue is to document it so that it isn't lost.

**Comment by shepazu** [comment]
Yes, and having thought about it a bit more, I think we should allow the user to swap axes at will. By having multiline display (navigated with the arrow keys, perhaps), we could show a wider range of values using the area chart, so we could allow for datasets with large minima and maxima. See #16 for more details.
